conf/php.ini

```diff
@@ -1,2 +1,7 @@
-memory_limit = PHP_MEMORY_LIMIT
+memory_limit = 1G
 date.timezone = "UTC"
+
+;; Zend Loader Configuration
+zend_extension = /usr/local/etc/php/zend/opcache.so
+zend_extension = /usr/local/etc/php/zend/ZendGuardLoader.so
+always_populate_raw_post_data = -1
```

Comment:
You can remove the `always_populate_raw_post_data ` line, this is deprecated as of php 5.6.0 http://php.net/manual/en/ini.core.php